src/main/java/frc/robot/subsystems/Climbers.java

```diff
@@ -103,7 +103,7 @@ public double getEncoderValue() {
     return ((getRightEncoderValue() + getLeftEncoderValue()) / 2); //Finds the average of the encoders.
   }
 
-  public void frictionBrakerOn(boolean on) {
+  public void FrictionBrakerOn(boolean on) {
```

Comment:
Can you put this in Camel Case? Camel Case is the proper convention. In methods, the first character should be lower case and all words after need to start with a capitalized letter. forExampleLikeThis.